Fix trailing comma description about quantifier, add description (#20377)
mdn/content: files/en-us/web/javascript/reference/trailing_commas/index.md

  - Trailing comma
browser-compat: javascript.grammar.trailing_commas
---

{{JsSidebar("More")}}

**Trailing commas** (sometimes called "final commas") can be useful when adding new elements, parameters, or properties to JavaScript code. If you want to add a new property, you can add a new line without modifying the previously last line if that line already uses a trailing comma. This makes version-control diffs cleaner and editing code might be less troublesome.

> JavaScript has allowed trailing commas in array literals since the beginning. Trailing commas are then gradually allowed in object literals, function parameters, named imports, named exports, and more.

Reviewer: ```suggestion
JavaScript has allowed trailing commas in array literals since the beginning. Trailing commas are now also allowed in object literals, function parameters, named imports, named exports, and more.
```
